README.md

```diff
@@ -35,38 +34,64 @@ There is also support for legacy XMPP protocol for FCM.
 
-6. There are 2 ways to communicate with the Firebase Cloud Messaging- XMPP and Admin SDK. At least one of these needs to be configured. To configure these, please look at the [FCM section](#fcm).
+6. The appserver uses the Admin SDK to communicate with the Firebase Cloud Messaging. To 
+   configure this, please look at the [FCM section](#fcm).
 
 7. To run the build on mac or linux, run the below -
    ```bash
     ./gradlew bootRun
    ```
    You can also run in an IDE (like IntelliJ Idea) by giving the `/src/main/java/org/radarbase/appserver/AppserverApplication.java` as the main class.
 
-8. The App-server is now connected to the FCM XMPP server and is able to send and receive messages. On your mobile application, try sending an upstream message using the FCM sdk for your platform. Notification scheduling parses payloads from upstream messages containing the action SCHEDULE. The format of the data payload of upstream message should contain at least-    
-   ```json
-    {
-     "data":
-         {
-             "notificationTitle":"Schedule id 1",
-             "notificationMessage":"hello",
-             "action":"SCHEDULE",
-             "time":"1531482349791",
-             "subjectId":"test",
-             "projectID":"test",
-             "ttlSeconds":"900"
-          },
-          ...
-     }
-   ```
+8. The App-server is now running and is able to send FCM messages. You can make the request to 
+   create a project, a user and a notification using the [REST API](#rest-api).
 
-9. Voila!, you will now receive a notification at the schedule time (specified by `time` in the payload) on your device.
+9. Voila!, you will now receive a notification at the schedule time (specified by `scheduledTime` in the payload) on your device.
 
-10. You can also achieve the same using more reliable and flexible REST API using the schedule endpoint. Please refer to REST API section below for more info.
+## REST API
 
-11. API documentation is available via Swagger UI when you launch the app server. Please refer to the Documentation section below.
+The full API specification and documentation is available via Swagger UI when you launch the app 
+server. Please refer to the [Documentation section](#documentation) below.
 
-## REST API
+1. Create a project. If using Management portal, this should be exactly same as the project name 
+   in management portal.
+    ```
+   POST http://localhost:8080/projects/p1
+   {
+    "projectId": "p1"
+   }
+   ```
+2. Create a user. Make sure to use the correct FCM token otherwise you will not receive the 
+   notification on the device.
+    ```
+   POST http://localhost:8080/projects/p1/users/u2
+   {
+    "subjectId": "u2",
+    "fcmToken" : "shdzdxcvc", 
+    "enrolmentDate": "2018-11-29T00:00:00Z",
+    "timezone": "Australia/Sydney",
+    "language": "en"
+    }
+   ```
+3. Create a notification.
```

Comment:
Should we add the endpoint here as well? `POST http://localhost:8080/projects/p1/users/u2/messaging/notifications` or `POST http://localhost:8080/projects/p1/users/u2/messaging/data`?